Symfony/CS/Fixer/Contrib/OrderedUseFixer.php

```diff
@@ -35,8 +35,7 @@ class OrderedUseFixer extends AbstractFixer
     private static $sortType = self::SORT_ALPHA;
 
     /**
-     * Array of supported sort types:
-     *  alpha, length.
+     * Array of supported sort types: alpha, length.
```

Comment:
Agree,  but @keradus asked: `whats' inside array ?`.  So I thought it should be. Or maybe I miss understud something